### PR TITLE
34/reduce noise p raul

### LINKED ISSUE #34: Reduce noise for PRaul: Automatic deployment notifications for PRs
**Current situation**
Currently, every PR deploys automatically the web and generates a test URL.
This happens on every commit.

Additionally it adds a comment to the PR providing the URL, so it's simpler to test.

**Problem**
This generates too much noise/SPAM, Github sends an email on every automatic deployment

### PR DESCRIPTION
**Pull request by W3stside**
Closes #34 

Adds JS script to check GH issue comments for 'gnosis-info' aka PRaul

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr129--dexreact.review.gnosisdev.com

File: travis/deploy_pull_request.sh [hunk 1]
```diff
@@ -26,7 +26,19 @@ function publish_pull_request_urls_in_github {
   # Done so because every PR is an issue, and the issues api allows to post general comments,
   # while the PR api requires that comments are made to specific files and specific commits
   GITHUB_PR_COMMENTS=https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments
-  curl -H "Authorization: token ${GITHUB_GNOSIS_INFO_API_TOKEN}" --request POST ${GITHUB_PR_COMMENTS} --data '{"body":"Travis automatic deployment:\r\n '${REVIEW_FEATURE_URL}'"}'
+  PREDICATE='gnosis-info'
+
+  # Check GITHUB_PR_COMMENTS if `gnosis-info` exists
+  # If present, do nothing as we want to reduce noise
+  # Else, comment URL
+  IS_COMMENT_PRESENT="$(node ./travis/check_pr_comments.js ${GITHUB_PR_COMMENTS} ${PREDICATE})"
```
Comment (Velenir) [inline travis/deploy_pull_request.sh]: While not an exact equivalent, you can use
```
IS_COMMENT_PRESENT=$(curl https://api.github.com/repos/gnosis/dex-react/issues/129/comments | grep -q '"login":\s*"gnosis-info"' && echo "true" || echo "false")
```
And skip JS and extra dependencies

Comment (W3stside) [reply travis/deploy_pull_request.sh]: I stick to what i know i guess haha

Comment (W3stside) [reply travis/deploy_pull_request.sh]: resolved in following commits

**Comment by fleupold** [comment]
Nice improvement!

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FVgRrC6b0XmksE%2Fgiphy.gif&token=much_improved" data-gitmeme-token="much_improved"></a>

Would be awesome, if PRaul could become a proper dependency so that fixes like this actually benefit other projects using this tool as well.

**Pull request merged**